neon-mmd/websurfx
# 📝 More `Websurfx` Instances with the `edge/rolling` version

## What does this PR do?
This pull request updates the list of Websurfx instances in the `instances.md` file. It adds new instances contributed by the community and updates the version to `v1.2.35`.




## Why is this change important?
The update is essential for maintaining an accurate and up-to-date list of Websurfx instances.




## How to test this PR locally?
1. Clone the repository.
2. Checkout the branch associated with this pull request.
3. Review the updated README file to ensure that the instances and version information are accurate.



## Author's checklist
- [x] Bumped the version to v1.2.35.


## Related issues
None
